src/components/PoliceNewsRoomCard.jsx

```diff
@@ -2,44 +2,48 @@ import {
   Card,
   CardContent,
   CardFooter,
-  Button
+  Button,
 } from "@baltimorecounty/dotgov-components";
 import PoliceNewsThumbnail from "./PoliceNewsThumbnail";
 import React from "react";
 
-const NewsRoomCard = props => {
+const NewsRoomCard = (props) => {
   const {
     publishDate,
     title,
     author,
     articleSummary,
     thumbnail,
     thumbnailAltText,
-    url
+    url,
   } = props;
 
   const options = {
     year: "numeric",
     month: "long",
-    day: "numeric"
+    day: "numeric",
   };
 
   const published = new Date(publishDate).toLocaleDateString("en-US", options);
 
+  const dg_police_news_date_author = {
```

Comment:
not blocking but this should probably be `authorStyles` or something like that.

Reply:
changed